src/ImageCarouselReact/widget/components/Carousel.tsx

```diff
@@ -191,22 +100,22 @@ class Carousel extends React.Component<ICarouselProps, ICarouselState> {
     public componentWillUnmount() {
         logger.debug(this.loggerNode + " .componentWillUnmount");
         clearTimeout(this.timeout);
-        this.isUnmounted = true; // TODO: Clear out isUnmounted
     }
 
     public render() {
         logger.debug(this.loggerNode + " .render");
         const props = this.props;
-        const { showIndicators, showControls, wrap, prevIcon, nextIcon, className, children, bsProps } = props;
+        const { showIndicators, showControls, wrap, prevIcon} = props;
+        const { nextIcon, className, children, bootstrapClass } = props;
         const activeIndex = this.getActiveIndex();
         let classes: any = {slide: this.props.slide};
```

Comment:
Can use const and literal object
const classes = {
            slide: this.props.slide,
            [bootstrapClass] : true
        };